pkg/apis/provisioning/v1alpha5/container_runtime_configuration.go

```diff
@@ -0,0 +1,61 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha5
+
+import (
+	"strings"
+
+	"knative.dev/pkg/apis"
+)
+
+// ContainerRuntimeConfiguration defines args to be used when configuring the Container Runtime.
+// Note, not all Container Runtime implementations will support all of these settings.
+type ContainerRuntimeConfiguration struct {
+	// RegistryMirrors a set of RegistryMirror configurations.
+	//+optional
+	RegistryMirrors []RegistryMirror `json:"registryMirrors,omitempty"`
+}
+
+// RegistryMirror configuration
+type RegistryMirror struct {
+	// The registry's domain name or "*" to match all registries.
+	Registry string `json:"registry,omitempty"`
+	// The endpoints to use as mirrors for that registry.
+	Endpoints []RegistryMirrorEndpoint `json:"endpoints,omitempty"`
```

Comment:
These two comments should be in the form

// Registry [verb] [explaination]

and 

// Endpoints [verb] [explaination]